src/Game.js

```diff
@@ -79,9 +101,16 @@ export default class Game {
     return this
   }
 
-  pause() {
-    this.isPaused = this.isPaused ? false : true
-  }
+  down() {
+    while (
+      !this.fallingPieceIsOutOfBounds() &&
```

Comment:
Consider refactoring this into an intention revealing method:
```js
while( this.pieceIsInValidPosition() ) {
  // etc
}

pieceIsInValidPosition() {
  return !this.fallingPieceIsOutOfBounds() && !this.fallingPieceOverlapsRubble()
}
```
On a related note, if it increases the readability of this code, [De Morgan's Law](https://en.wikipedia.org/wiki/De_Morgan's_laws) states that `! a && ! b` is equivalent to `!( a || b )`